# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/tabcrashed/file_iframe.html
# Commit: 8197bf1a1fed
# Full Hash: 8197bf1a1fed1f34234bdef3eaacd2e1d6a873d2
# Author: Anny Gakhokidze <agakhokidze@mozilla.com>
# Date: 2020-05-09 09:14:23
# Description:
#   Bug 1633471 - Do not reuse inner window when printing, r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74261
# ==============================================================================

diff -r 220a0d9666fe -r 8197bf1a1fed browser/base/content/test/tabcrashed/file_iframe.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/browser/base/content/test/tabcrashed/file_iframe.html	Fri May 08 20:13:16 2020 +0000
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta charset="utf-8">
+</head>
+<body>
+Iframe body
+</body>
+</html>